### PR TITLE
Provide an example on how to debug a web app.

### PR DESCRIPTION
**Pull request by jbuberel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,6 +65,34 @@ spew.Fprintf(someWriter, "myVar1: %v -- myVar2: %+v", myVar1, myVar2)
 spew.Fprintf(someWriter, "myVar3: %#v -- myVar4: %#+v", myVar3, myVar4)
 ```
 
+## Debugging a Web Application Example
+
+Here is an example of how you can use `spew.Sdump()` to help debug a web application. Please be sure to wrap your output using the `html.EscapeString()` function for safety reasons. You should also only use this debugging technique in a development environment, never in production.
+
+```
```
Comment (shurcooL) [inline README.md]: I think you should use Go highlighting here. It's already done above, and it makes the code more readable.

To do so, change \`\`\` into \`\`\`Go, like above.

**Comment by jbuberel** [comment]
Agreed- added the Go tag.

**Comment by davecgh** [comment]
Please rebase/squash this to a single commit.

File: README.md [hunk 2]
```diff
@@ -65,6 +65,34 @@ spew.Fprintf(someWriter, "myVar1: %v -- myVar2: %+v", myVar1, myVar2)
...
+```Go
+package main
+
+import (
+    "fmt"
+    "html"
+    "net/http"
+
+    "github.com/davecgh/go-spew/spew"
+
```
Comment (shurcooL) [inline README.md]: This blank line is unnecessary.

Comment (davecgh) [inline README.md]: True that's it's not necessary, but I'd prefer that it stays because that is how goimports formats it.

Comment (shurcooL) [inline README.md]: That's not true. I meant the blank line after `"github.com/davecgh/go-spew/spew"`, not before.

It should be:

```Go
import (
	"fmt"
	"html"
	"net/http"

	"github.com/davecgh/go-spew/spew"
)
```

Comment (davecgh) [inline README.md]: Oh, I thought I saw your comment on the line above it.  You're right of course.  :+1:

**Comment by jbuberel** [comment]
Have to admit that I don't know exactly how to do that (rebase/squash) with Git. Is it easier to abandon this pull request and resubmit a new one with single commit?

Suggestions?

**Comment by davecgh** [comment]
> $ git rebase -i master
> Change the second and third commits to 's' (per the codes shown in the editor and assuming you've removed the extra blank line in another commit)
> $ git push -f

**Comment by jbuberel** [comment]
I think I have it figured out now - sorry for the delay. PTAL.

**Comment by davecgh** [comment]
Thanks!

**Pull request merged**